README.md

```diff
@@ -82,7 +82,7 @@ It takes as children a single, argumentless function which should return exactly
 The rendering in the function will be tracked and automatically re-rendered when needed.
 This can come in handy when needing to pass render function to external components (for example the React Native listview), or if you
 dislike the `observer` decorator / function.
-
+And it can work with `Provider` use inject  and render property / or just takes as children, detail in Example2.(PS:using children has a priority than render , so dont use at the same time)
```

Comment:
I would split this, there are two concerns to cover - render prop and injecting. Also, I have a trouble understanding what it says. Let me rephrase this.

> In case you are a fan of render props, you can use that instead of children. Be advised, that you cannot use both approaches at once, children have a precedence.

_Here goes example showing render prop only_

> Observer can also inject the stores simply by passing a selector function.

_Second example with inject_

Reply:
Oh and with injecting it's worth to show an example that passing strings work same way as with `inject` HoC.